proof/refine/ARM/Reply_R.thy

```diff
@@ -1035,4 +1035,68 @@ lemma updateReply_replyNext_Nothing_invs':
                   dest: pspace_alignedD' pspace_distinctD')
   done
 
+context begin interpretation Arch .
+
+lemma pspace_relation_reply_update_conc_only:
+  "\<lbrakk> pspace_relation ps ps'; ps x = Some (Structures_A.Reply reply); ps' x = Some (KOReply reply');
+     reply_relation reply new\<rbrakk>
+   \<Longrightarrow> pspace_relation ps (ps'(x \<mapsto> (KOReply new)))"
```

Comment:
Out of curiosity, do you remember why this needed `Arch`?

Reply:
It think this is because of `arch_kernel_obj.split_asm` splitting and simplification around it at the end.